python/python_frontend/translation.cpp

```diff
@@ -1319,6 +1319,41 @@ class FusionTranslator : public OptInConstDispatch {
         topkop->isSorted()));
   }
 
+  // Helper function to get the corresponding name for ScanOp based on its
+  // BinaryOpType
+  std::string getScanOpString(const ScanOp* scan_op) {
```

Comment:
Why not use `NVF_API std::string toString(const ScanOp* sop);` and have assertion 
`NVF_ERROR(scan_op->opType() == BinaryOpType::Add, "ScanOp type not implemented: only Add (cumsum) is supported.");`?